pkg/deploy/deployment_che.go

```diff
@@ -112,10 +112,12 @@ func NewCheDeployment(cr *orgv1.CheCluster, cheImage string, cheTag string, cmRe
 										Scheme: corev1.URISchemeHTTP,
 									},
 								},
-								InitialDelaySeconds: 25,
-								FailureThreshold:    5,
+								InitialDelaySeconds: 60,
+								FailureThreshold:    10,
 								TimeoutSeconds:      5,
-							},
+								PeriodSeconds:       10,
+							},	// After POD start, the POD will be declared as either ready or failed after a minimum of 60 seconds and a maximum of 200 seconds
+									// 200 s = InitialDelaySeconds + TimeoutSeconds * FailureThreshold + PeriodSeconds * (FailureThreshold - 1)
```

Comment:
not sure if this is the intentional offset for the comment ;-)

Reply:
Well, it is tab-based indentation which, in VSCode at least, seems to align both comments.

Reply:
Should be fixed in commit a77fc44